README.md

```diff
@@ -19,12 +19,13 @@ This analysis repository presumes that the following python packages are availab
  * basemap
  * lxml
  * nco
+ * pyproj
```

Comment:
`pyproj` is a handy tool that @matthewhoffman showed me for defining logically rectangular projections.  This is useful for polar stereographic projections commonly used for data sets and visualization in polar regions.